app/src/main/res/layout/row_download.xml

```diff
@@ -41,6 +41,20 @@
 
     </LinearLayout>
 
+    <ImageButton
+        android:id="@+id/btn_copy_link"
+        android:layout_width="50dp"
+        android:layout_height="50dp"
+        style="@style/SelectableBorderlessItem"
+        android:layout_gravity="center_vertical"
+        android:contentDescription="@string/copy_download_link"
+        android:src="@drawable/icon_copy_link" />
```

Comment:
I think I'd rather add this as long.-press action. Long press for copying links is a convention I've often seen in other apps, and doing so makes the asset item less visually crowded.